Main.hs

```diff
@@ -1,38 +1,35 @@
-{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
-
-{-# HLINT ignore "Use foldr" #-}
-{-# HLINT ignore "Use map" #-}
-{-# HLINT ignore "Use product" #-}
-{-# HLINT ignore "Use sum" #-}
```

Comment:
Please do not remove these annotations. We have annotated the annotations with some purpose)